src/app/beer_garden/db/mongo/models.py

```diff
@@ -358,8 +360,21 @@ class Request(MongoModel, Document):
 
     logger = logging.getLogger(__name__)
 
+    def dumps(self, *args, **kwargs):
+        """If string output was over 16MB it was spilled over to the GridFS storage solution"""
+        if self.output_gridfs:
+            self.output = self.output_gridfs.read()
+            self.output_gridfs = None
```

Comment:
I really wish we had a way to tell if this has a significant impact on our db performance. I'm scared this is incurring an entire DB roundtrip for every request we load, which *could* be very bad, but I don't know that for sure.

Reply:
This does cause for a long delay when trying to pull down a single request. Rendering a file that large just takes a while.

When we load the Requests page we have no additional latency because the output field is not returned from the query. 

Can we think of any other times we load requests from the database.